generator/AWSPSGeneratorLib/HelpMaterials/Examples/ELB2/Edit-ELB2TargetGroupAttribute.xml

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$newAttribute = New-Object Amazon.ElasticLoadBalancingV2.Model.TargetGroupAttribute
+$newAttribute.Key = 'deregistration_delay.timeout_seconds'
+$newAttribute.Value = 600
+
+Edit-ELB2TargetGroupAttribute -TargetGroupArn 'arn:aws:elasticloadbalancing:us-east-1:123456789012:targetgroup/test-tg/a4e04b3688be1970' -Attribute $newAttribute -Region us-east-1
```

Comment:
Can be simplified to `-Attribute @{Key = 'deregistration_delay.timeout_seconds'; Value = 600}`